### PR TITLE
FISH-10019 Core Results

### PR DESCRIPTION
**Pull request by abdulrahim458**

### CODE REVIEW

**Review by Pandrex247 — APPROVED** [review]

File: community/docs/modules/ROOT/pages/Jakarta EE Certification/7.2024.1.Alpha3/7.2024.1.Alpha3 Core TCK Results.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for Payara Server releases for Jakarta EE Core Profile 10.
```
Comment (Pandrex247) [inline community/docs/modules/ROOT/pages/Jakarta EE Certification/7.2024.1.Alpha3/7.2024.1.Alpha3 Core TCK Results.adoc:4]:
```suggestion
As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for Payara Server releases for Jakarta EE Core Profile 11.
```

File: community/docs/modules/ROOT/pages/Jakarta EE Certification/7.2024.1.Alpha3/7.2024.1.Alpha3 Core TCK Results.adoc [hunk 2]
```diff
@@ -14,36 +14,29 @@ As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Tech
...
 
 - Specification Name, Version and download URL:
 +
-https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE Core Profile 10]
+https://jakarta.ee/specifications/coreprofile/11//[Jakarta EE Core Profile 11]
 - TCK Version, digital SHA-256 fingerprint and download URL:
 +
-https://download.eclipse.org/jakartaee/coreprofile/10.0/jakarta-core-profile-tck-10.0.0.zip[Jakarta EE Core Profile TCK 10.0.0.zip]
+https://download.eclipse.org/jakartaee/coreprofile/11.0/jakarta-core-profile-tck-11.0.0.zip[Jakarta EE Core Profile TCK 11.0.0.zip]
 , SHA-256: `702a84e3c6532fb0aad6a37443bf5e7fadbfb7cf0071271f3a1fdd10cab1229c`
```
Comment (Pandrex247) [inline community/docs/modules/ROOT/pages/Jakarta EE Certification/7.2024.1.Alpha3/7.2024.1.Alpha3 Core TCK Results.adoc:26]: This SHA needs updating

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

File: community/docs/modules/ROOT/pages/Jakarta EE Certification/7.2024.1.Alpha3/7.2024.1.Alpha3 Core TCK Results.adoc (hunk 2 above)
Comment (Pandrex247) [reply community/docs/modules/ROOT/pages/Jakarta EE Certification/7.2024.1.Alpha3/7.2024.1.Alpha3 Core TCK Results.adoc:26]: Staged URL: https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee11/staged/eftl/jakarta-core-profile-tck-11.0.0.zip
SHA-256: 0357bfab7025972edb2bf50277b6b4206b499a2961bc94e783f34782cc4a9bda

File: community/docs/modules/ROOT/pages/Jakarta EE Certification/7.2024.1.Alpha3/7.2024.1.Alpha3 Core TCK Results.adoc (hunk 1 above)
Comment (abdulrahim458) [reply community/docs/modules/ROOT/pages/Jakarta EE Certification/7.2024.1.Alpha3/7.2024.1.Alpha3 Core TCK Results.adoc]: updated to core 11

**Pull request merged**